src/reducer.js

```diff
@@ -1,15 +1,22 @@
 import { LOCATION_CHANGE } from './actions'
 
+const getSearchQuery = (location) => {
+  if (location && location.search) {
+    return location.search
+  }
+
+  if (window && window.location && window.location.search) {
```

Comment:
Since this package will be used outside of the browser environment, then it is a good idea to remove this part since it trying to check for `window` and this might or would result in an error in a Node environment. The first check is enough for this. Thank you.

Reply:
(regression, sorry)

Reply:
@KorobkaVadim, what I meant is removing the whole `if (window !== 'undefined' && window.location && window.location.search)` block. I do not think it is required.

Reply:
Are you sure? As I understand it, this allows you to take parameters from a link. This is still your code, it's up to you)

Reply:
Smth like that?
```
const injectQuery = (location) => {
  const searchQuery = location && location.search

  if (typeof searchQuery !== 'string' || searchQuery.length === 0) {
    return {
      ...location,
      query: {}
    }
  }
```

Reply:
Yes please. As at when I wrote that code, I thought the package was used exclusively in the browser.

Reply:
Done